src/NuGet.Core/NuGet.PackageManagement/PackageDownloader.cs

```diff
@@ -127,7 +127,8 @@ public static class PackageDownloader
                     {
                         if (isPackageNamespaceEnabled)
                         {
-                            if (configuredPackageSources != null &&
+                            if (configuredPackageSources == null ||
```

Comment:
Should we exit early after we get the configured sources instead of here?

Reply:
@nkolev92 
I think current one is enough for now. We need to wait before different kind of mode implementation, depending on how different type mode implementation this piece of code will evolve.

Also I committed fix for failing `install scenario` tests.